docs/userguides/projects.md

```diff
@@ -38,13 +38,22 @@ ape compile
 
 **NOTE**: Compiler plugins download missing compiler version binaries, based on the contracts' pragma-spec.
 
-The contract types are then accessible from the `project` manager; deploy them in the `console` or in scripts:
+The contract containers are then accessible from the `project` manager; deploy them in the `console` or in scripts:
 
 ```python
 from ape import accounts, project
 
-a = accounts.load("metamask_0")
-a.deploy(project.MyContract)
+account = accounts.load("metamask_0")
```

Comment:
can we use a different name than `"metamask_0"`? feel like that might be misleading that we support metamask